### PR TITLE
update locations seed data

### PR DESCRIPTION
**Pull request by davedwight**
# Description

This updates the locations seed data to include coordinates

## What work was done?

A script was written to get coordinates for all new addresses, new addresses were found manually via Google Maps, the script was run against these new addresses and the resulting locations are now stored in `addresses_data.js`. The locations seed file was refactored to grab the new locations and no longer uses faker to retrieve address information.

## Why was this work done?

The locations in the locations table did not have coordinates.

## Type of change

- Bug Fix

## Change Status

- Completed

## Checklist

- [ x ] Endpoint tested with Postman, or Unit test.
- [ x ] My code follows the style guidelines of this project
- [ x ] I have performed a self-review of my own code
- [ x ] My changes generate no new warnings
- [ x ] There are no merge conflicts

https://www.loom.com/share/6e76efeb15624e4cb7ba48bfedaa2fd1

Coauthored by: Murray Warnock murray.warnock@gmail.com
Coauthored by: Michael Quint michaelquintdev@gmail.com
Coauthored by: Salomon Maldonado escttsalo@gmail.com

### CODE REVIEW

File: api/utils/getAddressesWithCoords.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    });
+};
+
+const loopGetCoords = async () => {
+  const updatedAddresses = [];
+  for (const a of addresses) {
+    const res = await getCoords(a);
+    updatedAddresses.push(res);
+  }
+  console.log(updatedAddresses);
```
Comment (michaelquintdev) [inline api/utils/getAddressesWithCoords.js]: We can remove this

**Review by michaelquintdev — APPROVED** [review]
Tested on my local machine and is working. In terms of the for loops I would normally go for a forEach but it's not that big of a deal it's still easily readable for future backend engineers. Great work!

**Review by escttsalo — APPROVED** [review]
Tested on my local env. Nothing broken, has a little bit of slow down on seeding but that's to be expected with the amount of data being seeded, and will prolly show on the production build as well. Works great. Well done!

File: api/utils/getAddressesWithCoords.js (hunk 1 above)
Comment (davedwight) [reply api/utils/getAddressesWithCoords.js]: What are you referring to? The exports?

**Comment by davedwight** [comment]
> Tested on my local machine and is working. In terms of the for loops I would normally go for a forEach but it's not that big of a deal it's still easily readable for future backend engineers. Great work!

Nice! One thing about forEach and map vs a for loop... I discovered that forEach and map don't treat async / await as you'd expect. When using await inside one of those, it doesn't halt execution until the promise is fulfilled like you would normally expect. But when using a for loop it does. That was part of the reason I was struggling to not overwhelm the DS API with concurrent requests and changing to a for loop solved the problem.

File: api/utils/getAddressesWithCoords.js (hunk 1 above)
Comment (escttsalo) [reply api/utils/getAddressesWithCoords.js]: The console.log most likely.

**Comment by davedwight** [comment]
> Tested on my local env. Nothing broken, has a little bit of slow down on seeding but that's to be expected with the amount of data being seeded, and will prolly show on the production build as well. Works great. Well done!

nice!

File: api/utils/getAddressesWithCoords.js (hunk 1 above)
Comment (davedwight) [reply api/utils/getAddressesWithCoords.js]: oh right, yeah makes sense

Comment (davedwight) [reply api/utils/getAddressesWithCoords.js]: deleted

**Pull request merged**